dist/preview release/what's new.md

```diff
@@ -13,21 +13,26 @@
   - webXRExperienceHelper to setup a default XR experience ([TrevorDev](https://github.com/TrevorDev))
   - WebXREnterExitUI and WebXRManagedOutputCanvas classes to configure the XR experience ([TrevorDev](https://github.com/TrevorDev))

```

Comment:
did a checkout of what's new rather than merging the master branch to my local.

Reply:
Well we should not see the added strings then because it is impossible to verify that everything is alright :(

Reply:
okay.. will have to do a merge then